dspace/config/spring/api/requestitem.xml

```diff
@@ -8,25 +8,50 @@
            http://www.springframework.org/schema/context/spring-context-2.5.xsd"
     default-autowire-candidates="*Service,*DAO,javax.sql.DataSource">
 
+    <description>Strategies for determining who receives Request Copy emails.</description>
+
     <context:annotation-config /> <!-- allows us to use Spring annotations in beans -->
 
+    <!-- Select the implementation to be used. -->
+    <alias alias='org.dspace.app.requestitem.RequestItemAuthorExtractor'
+           name='org.dspace.app.requestitem.RequestItemMetadataStrategy'/>
+
+    <!-- Get recipients from an item metadata field. -->
     <bean class="org.dspace.app.requestitem.RequestItemMetadataStrategy"
-          id="org.dspace.app.requestitem.RequestItemAuthorExtractor"
-          autowire-candidate='true'>
-    <!--
-        Uncomment these properties if you want lookup in metadata the email and
+          id="org.dspace.app.requestitem.RequestItemMetadataStrategy"
+          autowire-candidate="true">
+	<!-- 
+		Uncomment these properties if you want lookup in metadata the email and
         the name of the author to contact for request copy.
-        If you don't configure that or if the requested item doesn't have these
+		If you don't configure that or if the requested item doesn't have these
         metadata, the submitter data are used as fail over.
 
-        <property name="emailMetadata" value="schema.element.qualifier" />
-        <property name="fullNameMetadata" value="schema.element.qualifier" />
-    -->
-    </bean>
+		<property name="emailMetadata"
+                  value="schema.element.qualifier" />
+		<property name="fullNameMetadata"
+                  value="schema.element.qualifier" />
+	-->	
+	</bean>
 
     <!-- HelpDesk to instead get RequestItem emails-->
-    <!--<bean class="org.dspace.app.requestitem.RequestItemHelpdeskStrategy"
-        id="org.dspace.app.requestitem.RequestItemAuthorExtractor"
-        autowire-candidate='true'></bean>-->
+    <bean class="org.dspace.app.requestitem.RequestItemHelpdeskStrategy"
+          id="org.dspace.app.requestitem.RequestItemHelpdeskStrategy"/>
+
+    <!-- List a Collection's administrators. -->
+    <bean class='org.dspace.app.requestitem.CollectionAdministratorsRequestItemStrategy'
+          id='org.dspace.app.requestitem.CollectionAdministratorsRequestItemStrategy'/>
+
+    <!-- Concatenate the results of other strategies. -->
+    <bean class='org.dspace.app.requestitem.CombiningRequestItemStrategy'
+          id='org.dspace.app.requestitem.CombiningRequestItemStrategy'
+          autowire='no'>
+        <constructor-arg>
+            <description>A list of references to RequestItemAuthorExtractor beans</description>
+            <list>
+                <ref bean='org.dspace.app.requestitem.RequestItemMetadataStrategy'/>
+                <ref bean='org.dspace.app.requestitem.CollectionAdministratorsRequestItemStrategy'/>
+            </list>
+        </constructor-arg>
+    </bean>
 
```

Comment:
I don't like the idea to have a couple of beans (potentially more with customisation) that live around if they are not needed.

Reply:
I've commented out the declarations that are not used.